scripts/ninja2dot.py

```diff
@@ -1,3 +1,6 @@
+""" This file generates a graph of the dependencies for the ninja build.
+"""
```

Comment:
In a multi-line docstring The opening triple quotes can either be on their own line, or with a summary line. However, I realize that this is a one-line docstring (consisting of only a summary line), so everything should be on one line.